actioncable/app/assets/javascripts/action_cable.coffee.erb

```diff
@@ -1,3 +1,4 @@
+#= export ActionCable
```

Comment:
Does this make blade a runtime requirement for anyone running out of a git clone?